collect_app/src/main/java/org/odk/collect/android/analytics/AnalyticsEvents.java

```diff
@@ -250,4 +250,9 @@ private AnalyticsEvents() {
      * Tracks how often form details with null or empty hashes are provided by a server
      */
     public static final String NULL_OR_EMPTY_FORM_HASH = "NullOrEmptyFormHash";
+
+    /**
+     * Tracks how often repeat groups are used in field-list groups
+     */
+    public static final String REPEATS_IN_FILED_LIST = "RepeatsInFieldListException";
```

Comment:
I don't think we need to track analytics for this because I don't think it will change our behavior. We know it's something some people try to do. At some point in some future we know we might want to have a table or otherwise condensed view of a repeat. Maybe that view will be used if a repeat is in a field-list or maybe that will still not be allowed.

Reply:
If we do choose to keep analytics, the constant name should be updated FILED -> FIELD